internal/controller/secrets/secrets.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package secrets
```

Comment:
Maybe we could use a more specific name for this package, go filename, and test filename?

Reply:
Renamed to `serviceaccounttokencleanup`